graphqlapi/src/test/java/io/stargate/graphql/schema/fetchers/dml/ScalarsDmlTest.java

```diff
@@ -185,28 +186,20 @@ public void incorrectLiteralsForScalarsShouldResultInError(Column.Type type, Obj
         arguments(Column.Type.Int, 0, null),
         arguments(Column.Type.Int, Integer.MAX_VALUE, null),
         arguments(Column.Type.Int, Integer.MIN_VALUE, null),
-        arguments(Column.Type.Smallint, (short) 0, null),
-        arguments(Column.Type.Smallint, (short) -1, null),
-        arguments(Column.Type.Smallint, (short) 1, null),
-        arguments(Column.Type.Smallint, Short.MAX_VALUE, null),
-        arguments(Column.Type.Smallint, Short.MIN_VALUE, null),
+        arguments(Column.Type.Smallint, 0L, null),
+        arguments(Column.Type.Smallint, -1L, null),
+        arguments(Column.Type.Smallint, 1L, null),
+        arguments(Column.Type.Smallint, (long) Short.MAX_VALUE, null),
+        arguments(Column.Type.Smallint, (long) Short.MIN_VALUE, null),
         arguments(Column.Type.Text, "abc123", "'abc123'"),
         arguments(Column.Type.Text, "", "''"),
         arguments(Column.Type.Time, "10:15:30.123456789", "'10:15:30.123456789'"),
-        arguments(Column.Type.Time, "13:45", "'13:45:00.000000000'"),
```

Comment:
This one didn't work anymore because `Coercing.serialize` now uses a specific format instead of just calling `LocalTime.toString()`.